.eslintrc.json

```diff
@@ -0,0 +1,42 @@
+{
+    "env": {
+      "browser": true,
+      "commonjs": true,
+      "es6": true,
+      "node": true,
+      "mocha": true
+    },
+    "extends": ["google"],
+    "globals": {
+      "Atomics": "readonly",
+      "SharedArrayBuffer": "readonly"
+    },
+    "ignorePatterns": [
+      "**/node_modules",
+      "**/sitespeed-result",
+      "CODEOWNERS",
+      "*.yml",
+      "*.md",
+      "package-lock.json"
+    ],
+    "overrides": [
+      {
+        "files": ["dist/*.js", "src/*.js", "demo/demo.js"],
+        "parserOptions": {
+          "sourceType": "module"
+        }
+      }
+    ],
+    "parserOptions": {
+      "ecmaVersion": 2020,
+      "sourceType": "module"
+    },
+    "plugins": [
+        "eslint-plugin-html"
+    ],
+    "rules": {
+      "require-jsdoc": "off",
+      "max-len": "off"
```

Comment:
Zouden we hier niet toch een max-len toepassen? Dit lijkt me wel beter voor de leesbaarheid.

Reply:
Als jij alle componenten daarvoor wil aanpassen wel ja :)

Reply:
Het is toch ook maar een goede code formatter toepassen? Dit lijkt me ook geen breaking change. Maar goed, misschien toch nog een andere ticket van maken dan?

Reply:
@Pieter-Beckers dit zal nog op de master toegevoegd worden. Dit zou normaal automatisch door de IDE toegepast kunnen worden en lost het formatting probleem bij reviewen op.

Reply:
> @Pieter-Beckers dit zal nog op de master toegevoegd worden. Dit zou normaal automatisch door de IDE toegepast kunnen worden en lost het formatting probleem bij reviewen op.

Blijkt toch niet zo eenvoudig te zijn, voorlopig revert ik terug naar `"max-len": "off"`. we kunnen dit eventueel later nog bekijken.